assemblyline/templates/_helpers.tpl

```diff
@@ -353,4 +390,43 @@ data:
 {{ end }}
 {{ end }}
 
+{{ if .Values.configuration.retrohunt.enabled }}
+  {{with merge (dict "host" "hauntedhouse") . }}
+    {{template "assemblyline.GenCert" .}}
+  {{end}}
+
+  {{with merge (dict "host" "hauntedhouse-worker") . }}
+    {{template "assemblyline.GenCert" .}}
+  {{end}}
+{{ end }}
```

Comment:
Would it be possible to conditionally add these to the list of hosts to generate certificates for in:
https://github.com/CybercentreCanada/assemblyline-helm-chart/blob/21f91e6d0d93aad5bde6c92b471f515da2bf14a3/assemblyline/templates/_helpers.tpl#L326 ?